src/parser.coffee

```diff
@@ -123,6 +127,8 @@ class VASTParser
                     # Get full URL if url is defined
                     ad.nextWrapperURL = @resolveVastAdTagURI(ad.nextWrapperURL, url)
 
+                 # sequence doesn't carry over in wrapper element
+                options.determinedSequence = ad.sequence
```

Comment:
What do you think about naming this `options.wrapperSequence`? It can be self explanatory and we can get rid of the comment above

Reply:
for sure, will also make a test